tests/test_input_damages.py

```diff
@@ -0,0 +1,1287 @@
+            },
+        )
+
+        d = os.path.join(tmp_path, "mortality_in")
+        if not os.path.exists(d):
+            os.makedirs(d)
+        infile = os.path.join(d, f"mortality_damages_batch{b}.zarr")
+
+        ds_in.to_zarr(infile)
+
+    prep_mortality_damages(
+        gcms=["ACCESS1-0", "GFDL-CM3"],
+        paths=[
+            os.path.join(tmp_path, f"mortality_in/mortality_damages_batch{b}.zarr")
+            for b in [6, 9]
+        ],
+        vars={
+            "delta_costs": "monetized_costs",
+            "delta_deaths": "monetized_deaths",
+            "histclim_deaths": "monetized_histclim_deaths",
+        },
+        outpath=os.path.join(tmp_path, "mortality_out"),
+        mortality_version=version_test,
+        path_econ=os.path.join(tmp_path, "econvars_for_test", "econvars_for_test.zarr"),
+    )
+
+    ds_out_actual = xr.open_zarr(
+        os.path.join(
+            tmp_path,
+            "mortality_out",
+            f"impacts-darwin-montecarlo-damages-v{version_test}.zarr",
+        )
+    )
+
+    ds_out_expected = xr.Dataset(
+        {
+            "delta": (
+                ["gcm", "batch", "ssp", "rcp", "model", "year", "region"],
+                np.full((2, 2, 2, 2, 2, 2, 2), -0.90681089),
+            ),
+            "histclim": (
+                ["gcm", "batch", "ssp", "rcp", "model", "year", "region"],
+                np.full((2, 2, 2, 2, 2, 2, 2), 2 * 0.90681089),
+            ),
+        },
+        coords={
+            "batch": (["batch"], ["batch6", "batch9"]),
+            "gcm": (["gcm"], ["ACCESS1-0", "GFDL-CM3"]),
+            "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+            "rcp": (["rcp"], ["rcp45", "rcp85"]),
+            "region": (["region"], ["USA.test_region", "ZWE.test_region"]),
+            "ssp": (["ssp"], ["SSP2", "SSP3"]),
+            "year": (["year"], [2010, 2099]),
+        },
+    )
+
+    xr.testing.assert_equal(ds_out_expected, ds_out_actual)
+
+
+def test_error_prep_mortality_damages(tmp_path):
+    """
+    Test that prep_mortality_damages complains when invalid mortality version is passed
+    """
+    with pytest.raises(ValueError) as excinfo:
+        prep_mortality_damages(
+            gcms=["ACCESS1-0", "GFDL-CM3"],
+            paths=[
+                os.path.join(tmp_path, f"mortality_in/mortality_damages_batch{b}.zarr")
+                for b in [6, 9]
+            ],
+            vars={
+                "delta_costs": "monetized_costs",
+                "delta_deaths": "monetized_deaths",
+                "histclim_deaths": "monetized_histclim_deaths",
+            },
+            outpath=os.path.join(tmp_path, "mortality_out"),
+            mortality_version=6,
+            path_econ=os.path.join(
+                tmp_path, "econvars_for_test", "econvars_for_test.zarr"
+            ),
+        )
+    assert "Mortality version not valid: " in str(excinfo.value)
+
+
+@pytest.mark.parametrize("version_test", ["v0.21", "v0.20"])
```

Comment:
Why don't we test v0.22?

Reply:
v0.22 has exactly the same structure as v0.21. Given that all values in the tests are fake anyway, it might be sufficient to test either v0.21 or v0.22 only.

Reply:
Makes sense. @davidrzhdu can you add a comment to that effect?

Reply:
This is resolved by https://github.com/ClimateImpactLab/dscim/pull/68/commits/7cf92eb818d8d6a997a3f74a9852386dfa3c1074